src/dotnet/APIView/APIViewWeb/Pages/Shared/_CommentThreadPartial.cshtml

```diff
@@ -30,22 +30,28 @@
                                     </button>
                                 </form>
 
-                                @if (comment.Username == User.GetGitHubLogin())
-                                {
-                                    <div class="dropdown show">
-                                        <a class="btn btn-sm" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                                            ...
-                                        </a>
+                                <div class="dropdown show">
+                                    <a class="btn btn-sm" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                        ...
+                                    </a>
 
-                                        <div class="dropdown-menu">
+                                    <div class="dropdown-menu clickable">
+                                        @if (comment.Username == User.GetGitHubLogin())
+                                        {
                                             <form data-post-update="comments" asp-controller="Comments" asp-action="Delete" method="post" asp-route-reviewId="@Model.ReviewId">
-                                                <input type="hidden" name="commentId" value="@comment.CommentId"/>
+                                                <input type="hidden" name="commentId" value="@comment.CommentId" />
                                                 <button id="@comment.CommentId" type="submit" class="text-danger dropdown-item">Delete</button>
                                             </form>
                                             <a class="dropdown-item js-edit-comment">Edit</a>
-                                        </div>
+                                        }
+                                        <a class="dropdown-item js-github" data-repo="azure-sdk-for-net" data-format="c#">Create issue in .NET repo</a>
```

Comment:
Shouldn't this also be done for authenticated github users?

Reply:
They have to authenticate to even see the page, but this is outside the check where we see if the comment.Username is the logged in user, since any ApiView user should be able to create an issue from a comment by any other user.

Reply:
I would love to see this working with the previously discussed feature where each review will have a language specified in it, so that we could just infer where to file the issue, by default, from that (whilst still supporting other repos as well)

Reply:
I think we can determine the repo for C#, Java, and Python, but for languages that upload a json file directly, we would need to include the Language node as you have mentioned.

For now I can look into updating it for these three languages. Since there may be times when you want to create an issue in multiple repos, we can list the review's language repo as the first in the list, when we know the review's language. Would that work?

Reply:
Yes, having the right default selected is the best approach imho

Reply:
Updated

Reply:
Reverted as per @pakrym